### ISSUE TITLE
Change to bulk submit negative results (maybe client side only)

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/liliservices#162]
We received this message from LIDE (skip below for tl;dr):

> I briefly looked at the Bulk Submit Negative Results button. Currently, all that the button does for me is pop up a window stating: Using this form assumes all sample replicates and negative controls for Extraction 1 of Analysis Batch 93 have a negative qPCR result. The way that's worded is slightly misleading or perhaps I didn't communicate  properly. All samples replicates and negative controls for a particular target replicates batch will be negative. Every sample replicate target batch within that AB will not be necessarily be negative nor is that what I was intending to communicate. The bulk load upload feature would be for each set of target replicates independently.

> For example if I extracted for the following targets within that batch: AdenoAF, AdenoCDF, TTR, and CS and each had 2 replicates, I would be able to able to use the data upload screen to upload a replicate file for one set of replicates for the CS and one set of replicates for AdenoAF. I would then be able to use the bulk load negative screen to upload the other set of replicates for the targets CS and AdenoAF and I would be able to use it to upload both set of replicates for targets TTR and AdenoCDF.

tl;dr: this seems to simply mean that the negative results must be submitted on a per-target replicate basis, not across the whole extraction batch. Currently, the client submits an array like the one below to `/pcrreplicatebatches/bulk_load_negatives/`, so I am thinking that the fix here is strictly a client-side one, as it can simply submit an array containing objects of one target and replicate at a time.  I don't think the serializer has any 'opinion' on the makeup of the array as long as all the required fields are present for each object. Am I correct on that assumption?

I thought _maybe_ the `/bulk_load_negatives/` was expecting the full extraction batch set of target-replicate combos, which would be the only reason a client side fix would not be the only action needed. 

```
[
  {
    "extraction_batch": 106,
    "target": 34,
    "replicate_number": 1,
    "pcr_pos_cq_value": 23
  },
  {
    "extraction_batch": 106,
    "target": 20,
    "replicate_number": 1,
    "pcr_pos_cq_value": 24
  },
  {
    "extraction_batch": 106,
    "target": 20,
    "replicate_number": 2,
    "pcr_pos_cq_value": 23
  },
  {
    "extraction_batch": 106,
    "target": 37,
    "replicate_number": 1,
    "pcr_pos_cq_value": 21
  },
  {
    "extraction_batch": 106,
    "target": 38,
    "replicate_number": 1,
    "pcr_pos_cq_value": 25
  },
  {
    "extraction_batch": 106,
    "target": 38,
    "replicate_number": 2,
    "pcr_pos_cq_value": 23
  },
  {
    "extraction_batch": 106,
    "target": 39,
    "replicate_number": 1,
    "pcr_pos_cq_value": 22
  },
  {
    "extraction_batch": 106,
    "target": 39,
    "replicate_number": 2,
    "pcr_pos_cq_value": 21
  },
  {
    "extraction_batch": 106,
    "target": 4,
    "replicate_number": 1,
    "pcr_pos_cq_value": 25
  },
  {
    "extraction_batch": 106,
    "target": 4,
    "replicate_number": 2,
    "pcr_pos_cq_value": 21
  },
  {
    "extraction_batch": 106,
    "target": 36,
    "replicate_number": 1,
    "pcr_pos_cq_value": 22
  },
  {
    "extraction_batch": 106,
    "target": 36,
    "replicate_number": 2,
    "pcr_pos_cq_value": 23
  }
]
```
